app/api/dao/task.py

```diff
@@ -1,13 +1,14 @@
 from datetime import datetime
 
+from app.utils.decorator_utils import email_verification_required
```

Comment:
This should come before `from app.utils.enum_utils import MentorshipRelationState`.